examples/internlm/README.md

```diff
@@ -0,0 +1,428 @@
+# InternLM
+
+This document shows how to build and run InternLM 7B / 20B models in TensorRT-LLM on both single GPU, single node multi-GPU and multi-node multi-GPU.
```

Comment:
InternLM ->InternLM chat. Is it more accurate?

Reply:
I think we don't need too much details here. Models without chat are also supported by TRTLLM. They have same structures, just with different weight trained on different data, and used in different scenarios. 

To make users aware of the difference, I have mentioned the differences and post detailed URLs in the next paragraph. So users following the README should found the correct one. 

There are also base models and chat models for LLama v2, but we just call them all LLama 2.

Reply:
make sense